code/iaas/model/src/main/java/io/cattle/platform/core/cleanup/TableCleanUp.java

```diff
@@ -37,6 +39,19 @@ public void cleanUp(){
                 .execute();
 
         long endProcessInstance = System.currentTimeMillis();
+
+        long startProcessExecution = System.currentTimeMillis();
+        long lowestProcessInstance = create()
```

Comment:
Only way to determine which process executions to remove. as we don't want to delete the ones where there process instances are still alive.